__tests__/app.test.js

```diff
@@ -34,4 +34,32 @@ describe("GET /api", () => {
                 expect(response.body.endpoints).toHaveProperty("GET /api/topics"); 
             });
     });
-});
\ No newline at end of file
+});
+
+/*
+similarily to ticket 2, but modified for the specifics of the question,
+we could do something along the lines of first of, expecting the length of the object to be 8 or more,
+as we're working with 8 parameters, then we could just do 8 different expect toHaveProperty's for each one we was asked to include.
+That overall should cover everything we're looking for?
+*/
+
+describe("GET article by id", ()=>{
+    test("",()=>{
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((responce) => {
+            const article = responce.body.article_id[0];
+            expect(article).toHaveProperty("author")
+            expect(article).toHaveProperty("title")
+            expect(article).toHaveProperty("article_id")
+
+            expect(article).toHaveProperty("body")
+            expect(article).toHaveProperty("topic")
+
+            expect(article).toHaveProperty("created_at")
+            expect(article).toHaveProperty("votes")
+            expect(article).toHaveProperty("article_img_url")
+        })
+    })
+})
```

Comment:
Good job handling the "happy path" here, but what about the "sad path" (error handling)?

We have 2x untested behaviours here:

- Status 400, invalid ID, e.g. string of "not-an-id"
- Status 404, non existent ID, e.g. 0 or 9999